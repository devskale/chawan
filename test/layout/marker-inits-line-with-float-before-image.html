<!DOCTYPE html>
<div style="display: flow-root">
<ul style="margin: 1em 0 0">
<li style="float: left"><a href="https://example.org">test</a></li>
</ul>
<ul style="margin: 1em 0 0">
<li>
<img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAIAAAD/gAIDAAAACXBIWXMAAC4jAAAuIwF4pT92AAAAkklEQVR42u3QoQ0AAAgDsFn+F7y7F3CYJr2gyQ5XCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyfpW2ApraeX08pUAAAAASUVORK5CYII=">
</ul>
</div>

<div style="width: 20ch; display: flow-root">
<div style="margin: 1em 0 0">
<li style="float: left; margin-left: 2ch"><a href="https://example.org">test</a>
</div>
<div style="float: right">
<a href="https://example.org">
<img align=top src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAGQAAABkCAIAAAD/gAIDAAAACXBIWXMAAC4jAAAuIwF4pT92AAAAkklEQVR42u3QoQ0AAAgDsFn+F7y7F3CYJr2gyQ5XCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyZIlSxYKZMmSJUsWCmTJkiVLFgpkyfpW2ApraeX08pUAAAAASUVORK5CYII=">
a
