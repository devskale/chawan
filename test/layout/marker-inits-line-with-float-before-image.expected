
  â€¢ [33mtest           [39m
                   
                   
                   
                   


â€¢ [33mtest             a[39m
                  
                  
                  
                  
